Box.V2/Converter/BoxItemConverter.cs

```diff
@@ -166,6 +166,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             // Load JObject from stream
             JObject jObject = JObject.Load(reader);
+            // The notification_email field for the user object is an object when a value exists and is an empty array when it isn't. The code below converts the array to null to avoid deserialization issues.  
```

Comment:
```suggestion
            // The notification_email field for the user object is an object when a value exists and is an empty array when it isn't. The code below converts the empty array to a null value to avoid deserialization issues.  
```